regression-test/suites/schema_change_p0/test_add_keys_light_schema_change.groovy

```diff
@@ -0,0 +1,454 @@
+                    `date` DATEV2 NOT NULL COMMENT "数据灌入日期时间",
+                    `city` VARCHAR(20) COMMENT "用户所在城市",
+                    `age` SMALLINT COMMENT "用户年龄",
+                    `sex` TINYINT COMMENT "用户性别",
+
+                    `cost` BIGINT SUM DEFAULT "0" COMMENT "用户总消费",
+                    `max_dwell_time` INT MAX DEFAULT "0" COMMENT "用户最大停留时间",
+                    `hll_col` HLL HLL_UNION NOT NULL COMMENT "HLL列",
+                    `bitmap_col` Bitmap BITMAP_UNION NOT NULL COMMENT "bitmap列")
+                AGGREGATE KEY(`user_id`, `date`, `city`, `age`, `sex`) DISTRIBUTED BY HASH(`user_id`)
+                BUCKETS 4
+                PROPERTIES ( "replication_num" = "1", "light_schema_change" = "true");
+            """
+        sql """ 
+                INSERT INTO add_keys_light_schema_change VALUES
+                (1, '2017-10-01', 'Beijing', 10, 1, 1, 20, hll_hash(1), to_bitmap(1))
+            """
+        sql """
+            ALTER TABLE add_keys_light_schema_change ADD COLUMN new_key_column INT default "2" AFTER user_id PROPERTIES ("timeout"="604800")
+            """
+        
+        def jobStateResult = waitJobFinish(""" SHOW ALTER TABLE COLUMN WHERE IndexName='add_keys_light_schema_change' ORDER BY createtime DESC LIMIT 1 """, 9)
+        assertNotEquals(jobStateResult[0][8], "-1")
+
+        qt_sc_11 """ select * from add_keys_light_schema_change order by user_id """
+
+        sql """
+            ALTER TABLE add_keys_light_schema_change ADD COLUMN new_mv_key2 INT default "2"
+            """
+
+        jobStateResult = waitJobFinish(""" SHOW ALTER TABLE COLUMN WHERE IndexName='add_keys_light_schema_change' ORDER BY createtime DESC LIMIT 1 """, 9)
+        assertEquals(jobStateResult[0][8], "-1")
+
+        // case 1.2: light schema change type check: add key column with enable_unique_key_merge_on_write
+        sql """ DROP TABLE IF EXISTS add_keys_light_schema_change """
+        sql """
+                CREATE TABLE IF NOT EXISTS add_keys_light_schema_change (
+                    `user_id` LARGEINT NOT NULL COMMENT "用户id",
+                    `date` DATEV2 NOT NULL COMMENT "数据灌入日期时间",
+                    `city` VARCHAR(20) COMMENT "用户所在城市",
+                    `age` SMALLINT COMMENT "用户年龄",
+                    `sex` TINYINT COMMENT "用户性别",
+
+                    `cost` BIGINT DEFAULT "0" COMMENT "用户总消费")
+                UNIQUE KEY(`user_id`, `date`, `city`, `age`, `sex`) DISTRIBUTED BY HASH(`user_id`)
+                BUCKETS 4
+                PROPERTIES ( 
+                    "replication_num" = "1", 
+                    "light_schema_change" = "true", 
+                    "enable_unique_key_merge_on_write" = "true");
+            """
+        sql """ 
+                INSERT INTO add_keys_light_schema_change VALUES
+                (1, '2017-10-01', 'Beijing', 10, 1, 20)
+            """
+        qt_sc_12 """ select * from add_keys_light_schema_change order by user_id """
```

Comment:
The test case should follow this process:
1. insert values
2. add key column with default value
3. insert values
4. check select value